lib/api/src/index.tsx

```diff
@@ -124,7 +125,7 @@ type StatePartial = Partial<State>;
 export type ManagerProviderProps = Children & RouterData & ProviderData & DocsModeData;
 
 class ManagerProvider extends Component<ManagerProviderProps, State> {
-  api: API;
+  api: API = {} as API;
```

Comment:
@tmeasday This is the change in API you needed before to gain access to fullAPI in the modules subAPI.